docs/examples/python_sdk_examples.ipynb

```diff
@@ -0,0 +1,355 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "isvc = V1beta1InferenceService(\n",
+    "    api_version=constants.KSERVE_V1BETA1,\n",
+    "    kind=constants.KSERVE_KIND,\n",
+    "    metadata=client.V1ObjectMeta(\n",
+    "        name=name, \n",
+    "        namespace=namespace,\n",
+    "        annotations={\n",
+    "            'serving.kserve.io/deploymentMode': 'ModelMesh'\n",
+    "        }\n",
+    "    ),\n",
+    "    spec=V1beta1InferenceServiceSpec(\n",
+    "        predictor=V1beta1PredictorSpec(\n",
+    "            sklearn=V1beta1SKLearnSpec(\n",
+    "                protocol_version=protocol_version,\n",
+    "                storage=V1beta1StorageSpec(\n",
+    "                    key='localMinIO',\n",
+    "                    path='sklearn/mnist-svm.joblib'\n",
+    "                )\n",
+    "            )\n",
+    "        )\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create `InferenceService`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "kserve = KServeClient()\n",
+    "isvc = kserve.create(isvc)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "name:  mnist-sample\n",
+      "uid:  45aa2662-4507-401e-98fa-aa78e3cea14e\n",
+      "key:  localMinIO\n",
+      "path:  sklearn/mnist-svm.joblib\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"name: \", isvc[\"metadata\"][\"name\"])\n",
+    "print(\"uid: \", isvc[\"metadata\"][\"uid\"])\n",
+    "print(\"key: \", isvc[\"spec\"][\"predictor\"][\"sklearn\"][\"storage\"][\"key\"])\n",
+    "print(\"path: \", isvc[\"spec\"][\"predictor\"][\"sklearn\"][\"storage\"][\"path\"])"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Check `InferenceService` status after deploying\n",
+    "It will be in a `Pending` state while the associated serving runtime pods start up."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "deployed_isvc = kserve.get(name, namespace=namespace)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Pending\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(deployed_isvc[\"status\"][\"modelStatus\"][\"states\"][\"activeModelState\"])"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Check `InferenceService` status once its ready\n",
+    "It should now be in a `Loaded` state."
```

Comment:
Even though we mentioned it above, I think we could explain when "now" is:

"Once the serving runtime pods are started, the `InferenceService` should be in `Loaded` state."